$OpenBSD: patch-Rakefile,v 1.1 2011/11/17 16:08:01 jeremy Exp $

Comment out requiring newgem hoe plugin, as it is not needed for
building and breaks regress.

--- Rakefile.orig	Sun Nov 13 00:22:57 2011
+++ Rakefile	Sun Nov 13 00:23:10 2011
@@ -4,7 +4,7 @@ require 'hoe'
 require 'fileutils'
 require './lib/templater'
 
-Hoe.plugin :newgem
+# Hoe.plugin :newgem
 # Hoe.plugin :website
 # Hoe.plugin :cucumberfeatures
 
@@ -19,5 +19,5 @@ $hoe = Hoe.spec 'templater' do
   self.extra_dev_deps << ['rspec', '>= 1.2.8']
 end
 
-require 'newgem/tasks'
+# require 'newgem/tasks'
 Dir['tasks/**/*.rake'].each { |t| load t }
